### ISSUE TITLE
NameError: name 'mappings_helper' is not defined

### ISSUE THREAD

**Issue by wolfcod** [mentat-is/gulp#9]
Following the documentation, I tried to ingest in gulp the data available in `./samples` directory as mentioned [here](https://github.com/mentat-is/gulp/blob/develop/docs/Install%20Dev.md#1-test).

Gulp log reports the following message:
```
024-09-19 09:45:01,989|gulp|Task-114|INFO|4930,4930|gulp/api/collab/stats.py,get(),731|---> get: found 1 stats: [GulpStats(id=3, type=5, req_id=cecde61a-b597-4974-b24e-0c69eb11979c, operation_id=1,             client_id=1, context=testcontext, status=1, time_created=1726731901461,             time_expire=1726818301461, time_update=1726731901966, time_end=1726731901966,             ev_failed=0, ev_skipped=0, ev_processed=0, parser_failed=24,             files_processed=24, files_total=24,             queries_total=0, queries_processed=0, matches_total=0,             ingest_errors={'/home/gulp/repo/gulp/samples/win_evtx/issue_201.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/2-system-Security-dirty.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/2-vss_0-Microsoft-Windows-TerminalServices-RemoteConnectionManager%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Security_short_selected.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/sample-with-irregular-bool-values.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/security.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/system.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/2-vss_0-Microsoft-Windows-RemoteDesktopServices-RdpCoreTS%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/2-vss_7-System.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Archive-ForwardedEvents-test.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/E_Windows_system32_winevt_logs_Microsoft-Windows-CAPI2%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/MSExchange_Management_wec.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/sample_with_a_bad_chunk_magic.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/E_Windows_system32_winevt_logs_Microsoft-Windows-Shell-Core%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/post-Security.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/new-user-security.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/2-system-Microsoft-Windows-LiveId%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Microsoft-Windows-LanguagePackSetup%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/security_big_sample.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Microsoft-Windows-HelloForBusiness%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Application.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/E_ShadowCopy6_windows_system32_winevt_logs_Microsoft-Windows-CAPI2%4Operational.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/Application_no_crc32.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"], '/home/gulp/repo/gulp/samples/win_evtx/sysmon.evtx': ["[/home/gulp/repo/gulp/src/gulp/plugins/win_evtx.py:ingest:320] NameError: name 'mappings_helper' is not defined\n"]},             current_src_file=/home/gulp/repo/gulp/samples/win_evtx/sysmon.evtx)]
2024-09-19 09:45:01,989|gulp|Task-114|ERROR|4930,4930|gulp/workers.py,_print_debug_ingestion_stats(),98|(NOT AN ERROR) req_id=cecde61a-b597-4974-b24e-0c69eb11979c, started_at=2024-09-19T07:45:01.461000+00:00, took seconds=0.505000, (minutes=0.008417, hours=0.000140)
final stats req_id=cecde61a-b597-4974-b24e-0c69eb11979c: processed=0, failed=0, skipped=0, num (UNIQUE) ingest_errors=24, QueuePool status=Pool size: 5  Connections in pool: 1 Current Overflow: -4 Current Checked out connections: 0
```

Trying to connect using `gulpui-web` the web application enter in an infinite loop, no longer accepting any kind of user interaction, and gulp server continue to print the same messages:
```
NFO:     127.0.0.1:56968 - "GET /query_operations?index=testidx HTTP/1.1" 200 OK
2024-09-19 09:56:57,438|gulp|Task-1648|DEBUG|4930,4930|gulp/api/collab/session.py,check_token(),377|---> check_token: token=452209bd-1c75-4d5c-baf4-c62f196aa677
2024-09-19 09:56:57,439|gulp|Task-1648|DEBUG|4930,4930|gulp/api/collab/session.py,get(),125|---> get: token=452209bd-1c75-4d5c-baf4-c62f196aa677
2024-09-19 09:56:57,440|gulp|Task-1649|DEBUG|4930,4930|gulp/api/collab/session.py,check_token(),377|---> check_token: token=452209bd-1c75-4d5c-baf4-c62f196aa677
2024-09-19 09:56:57,440|gulp|Task-1649|DEBUG|4930,4930|gulp/api/collab/session.py,get(),125|---> get: token=452209bd-1c75-4d5c-baf4-c62f196aa677
2024-09-19 09:56:57,441|gulp|Task-1648|DEBUG|4930,4930|gulp/api/collab/session.py,get(),133|get: user_session for token 452209bd-1c75-4d5c-baf4-c62f196aa677=UserSession(id=2, user_id=1, token='452209bd-1c75-4d5c-baf4-c62f196aa677', time_expire=1726733206683)
2024-09-19 09:56:57,442|gulp|Task-1649|DEBUG|4930,4930|gulp/api/collab/session.py,get(),133|get: user_session for token 452209bd-1c75-4d5c-baf4-c62f196aa677=UserSession(id=2, user_id=1, token='452209bd-1c75-4d5c-baf4-c62f196aa677', time_expire=1726733206683)
2024-09-19 09:56:57,448|gulp|Task-1649|DEBUG|4930,4930|gulp/api/elastic_api.py,query_max_min_per_field(),892|query_max_min_per_field: q={
  "query": {
    "query_string": {
      "query": "(operation_id: 1)",
      "analyze_wildcard": true,
      "default_field": "_id"
    }
  }
}
2024-09-19 09:56:57,450|gulp|Task-1648|WARNING|4930,4930|gulp/api/elastic_api.py,query_operations(),1073|no results found, returning empty aggregations (possibly no data on elasticsearch)!
2024-09-19 09:56:57,454|gulp|Task-1648|DEBUG|4930,4930|gulp/api/rest/query.py,query_operations_handler(),1143|query_operations (before parsing): {
  "total": 0,
  "aggregations": {
    "operations": {
      "doc_count_error_upper_bound": 0,
      "sum_other_doc_count": 0,
      "buckets": []
    }
  }
}
2024-09-19 09:56:57,459|gulp|Task-1648|DEBUG|4930,4930|gulp/api/collab/operation.py,get(),200|---> get: filter=None
2024-09-19 09:56:57,459|muty|Task-1649|INFO|4930,4930|muty/jsend.py,success_jsend(),128|{
  "status": "success",
  "timestamp_msec": 1726732617459,
  "req_id": "dada1c36-0c10-41bb-9c86-2085834f1b45",
  "data": {
    "total": 0,
    "aggregations": {}
  }
}
```

**Comment by calligraf0** [comment]
Seems like this was a typo of instead of `mapping_helpers`, it was called as `mappings_helper`. :) 
Should be fixed now, try again with latest commit ([f44432d](https://github.com/mentat-is/gulp/commit/f44432db8fab7fa64e9abe33cbcff44a1a8988c8))

**Comment by wolfcod** [comment]
thanks. I tried and now it's working.
